Shopping.Cart.Tests/CartClearedTests.cs

```diff
@@ -20,6 +20,6 @@ public async Task ClearCartTest()
             cartId
         ));        
         
-        Assert.That(uncommittedEvents[2], Is.TypeOf<CartClearedCommand>());
+        Assert.That(uncommittedEvents[0], Is.TypeOf<CartClearedCommand>());
```

Comment:
`ClearCartCommandHandler.Handle` only emits one uncommitted event in this case so the `[2]` indexer is incorrect and fails the test.  This fixes that.